### PR TITLE
Simplify Readme

### LINKED ISSUE #514: Simplify README
Inline docs should be removed form the README. Instead, the README should point to `https://lit-element.polymer-project.org/`.

The examples should also be updated to use decorators.

### PR DESCRIPTION
**Pull request by sorvell**
Fixes #514 

* removes inline docs and points to web site docs
* refines examples

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,289 +1,83 @@
 # LitElement
+A simple base class for creating fast, lightweight web components.
```
Comment (justinfagnani) [inline README.md]: "...with lit-html"?

File: README.md [hunk 2]
```diff
@@ -1,289 +1,83 @@
...
 [![Published on npm](https://img.shields.io/npm/v/lit-element.svg)](https://www.npmjs.com/package/lit-element)
 [![Published on webcomponents.org](https://img.shields.io/badge/webcomponents.org-published-blue.svg)](https://www.webcomponents.org/element/lit-element)
 [![Mentioned in Awesome lit-html](https://awesome.re/mentioned-badge.svg)](https://github.com/web-padawan/awesome-lit-html)
 
-## A simple base class for creating fast, lightweight web components
+## Documentation
+
+Full documentation is available at [lit-element.polymer-project.org](https://lit-element.polymer-project.org).
+
+Docs source is in the `docs` folder. To build the site youself, see the instructions in [docs/README.md](docs/README.md).
```
Comment (justinfagnani) [inline README.md]: I think we should move this into CONTRIBUTING.md for both projects

File: README.md [hunk 3]
```diff
@@ -1,289 +1,83 @@
...
+      // This decorator creates a property accessor that triggers rendering and
+      // an observed attribute.
+      @property()
+      mood = 'great';
 
-      constructor() {
-        super();
-        this.mood = 'happy';
-      }
+      // Provide styling for the element.
```
Comment (justinfagnani) [inline README.md]: maybe too many comments...

File: README.md [hunk 4]
```diff
@@ -1,289 +1,83 @@
...
-  1. Then implement a `render()` method and use the element's
-current properties to return a `lit-html` template result to render
-into the element.
-
-```html
-  <script src="node_modules/@webcomponents/webcomponentsjs/webcomponents-bundle.js"></script>
-  <script type="module">
-    import {LitElement, html} from 'lit-element';
+```ts
+    import {LitElement, html, css} from 'lit-element';
```
Comment (justinfagnani) [inline README.md]: import customElement and property

File: README.md [hunk 5]
```diff
@@ -1,289 +1,83 @@
...
+      // an observed attribute.
+      @property()
+      mood = 'great';
 
-      constructor() {
-        super();
-        this.mood = 'happy';
-      }
+      // Provide styling for the element.
+      static styles = css`.mood { color: green; }`;
```
Comment (justinfagnani) [inline README.md]: I'd add a newline after `

Comment (justinfagnani) [inline README.md]: Why not use an id as a selector to show of that that works in shadow dom? also class and property both named "mood" is maybe confusing.

File: README.md [hunk 6]
```diff
@@ -1,289 +1,83 @@
...
-      <h4>Foo: ${this.foo}</h4>
-      <div>whales: ${'🐳'.repeat(this.whales)}</div>
-      <slot></slot>
-    `;
-  }
+## Installation
 
-}
-customElements.define('my-element', MyElement);
+```bash
```
Comment (justinfagnani) [inline README.md]: From inside your project folder, run:

File: README.md (hunk 1 above)
Comment (sorvell) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (sorvell) [reply README.md]: Done.

File: README.md (hunk 3 above)
Comment (sorvell) [reply README.md]: Removed.

File: README.md (hunk 4 above)
Comment (sorvell) [reply README.md]: Added.

File: README.md (hunk 5 above)
Comment (sorvell) [reply README.md]: Done

Comment (sorvell) [reply README.md]: Simplified

File: README.md (hunk 6 above)
Comment (sorvell) [reply README.md]: Done.

File: README.md [hunk 7]
```diff
@@ -1,289 +1,85 @@
...
-element is not pending another update, and any code awaiting the element's
-`updateComplete` Promise runs and observes the element in the updated state.
-
-## Bigger Example
+```html
+    <my-element mood="awesome"></my-element>
+```
 
 Note, this example uses decorators to create properties. Decorators are a proposed
 standard currently available in [TypeScript](https://www.typescriptlang.org/) or [Babel](https://babeljs.io/docs/en/babel-plugin-proposal-decorators).
```
Comment (kevinpschaaf) [inline README.md]: Add to this paragraph:

> `LitElement` also supports a vanilla JavaScript method of declaring reactive properties.  

(maybe with a link to the docs section for that?)

**Pull request merged**